flink-connector-influxdb2/src/test/java/org/apache/flink/streaming/connectors/InfluxDBSourceIntegrationTestCase.java

```diff
@@ -67,59 +71,73 @@
 
     private final StreamExecutionEnvironment env =
             StreamExecutionEnvironment.getExecutionEnvironment().setParallelism(1);
-    private final InfluxDBSource<Long> influxDBSource =
-            InfluxDBSource.<Long>builder()
-                    .setPort(PORT)
-                    .setDeserializer(new InfluxDBTestDeserializer())
-                    .build();
 
     @BeforeAll
-    static void setUp() {
+    void setUp() {
         CollectSink.VALUES.clear();
+        try {
+            this.port = new ServerSocket(0).getLocalPort();
```

Comment:
This is still using the same port again.
![image](https://user-images.githubusercontent.com/32157576/111035724-fc7d8500-841b-11eb-8e6c-df3a7287c708.png)

Reply:
Does not fix the problem